doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -127,6 +127,13 @@ nRF Desktop
    * Config channel no longer uses orphaned sections to store module Id information.
      Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
 
+Thingy:53: Matter weather station
+---------------------------------
+
+* Added:
+
+  * Initial support for the Matter-compliant over-the-air (OTA) device firmware upgrade (DFU) method.
+
 Samples
 =======
```

Comment:
The addition to samples should be in the changelog as well, otherwise everything looks good to me.

Reply:
Ok, done.